docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 # Docker-compose definition for development
 services:
   db:
-    image: postgres:14
+    image: postgres:16
```

Comment:
😢  Maybe let's not? Or what would be the arguments for changing the version again?

Reply:
Oh, that was an accidental commit because I have to do this every time to make my local setup work 😅